huaweicloud/services/cts/resource_huaweicloud_cts_notification.go

```diff
@@ -277,6 +336,13 @@ func resourceCTSNotificationRead(_ context.Context, d *schema.ResourceData, meta
 		)
 	}
 
+	if ctsNotification.Filter != nil {
+		filter := flattenNotificationFilter(ctsNotification.Filter)
+		mErr = multierror.Append(mErr,
+			d.Set("filter", filter),
+		)
+	}
```

Comment:
```
func flattenNotificationFilter(filter *cts.Filter) []map[string]interface{} {
	if filter == nil {
		return nil
	}
```
that you can put your code `d.Set("filter", flattenNotificationFilter(ctsNotification.Filter))` into the multierror append method in line 335

Reply:
fixed, pls check.